pkg/storage/export/export/export.go

```diff
@@ -1391,7 +1392,7 @@ func (ctrl *VMExportController) expandVirtualMachine(vm *virtv1.VirtualMachine)
 		return vm, nil
 	}
 
-	conflicts := ctrl.instancetypeMethods.ApplyToVmi(field.NewPath("spec", "template", "spec"), instancetypeSpec, preferenceSpec, &vm.Spec.Template.Spec, &vm.Spec.Template.ObjectMeta)
+	conflicts := instancetypeapply.NewVMIApplier().ApplyToVMI(field.NewPath("spec", "template", "spec"), instancetypeSpec, preferenceSpec, &vm.Spec.Template.Spec, &vm.Spec.Template.ObjectMeta)
```

Comment:
I'll handle this in a different PR, just focus on the unit tests for now.

Reply:
I will remove it, when the #13796 is merged.